rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -460,6 +460,18 @@ public enum BridgeMethods {
         fixedPermission(true),
         CallTypeHelper.ALLOW_STATIC_CALL
     ),
+    GET_STATE_FOR_SVP_CLIENT(
+        CallTransaction.Function.fromSignature(
+                "getStateForSvpClient",
+                new String[]{},
+                new String[]{"bytes"}
+        ),
+        fixedCost(4000L), // TODO: check fixed cost value
```

Comment:
i think there's no need to leave a TODO since we already have the task to address this created. Wdyt?

Reply:
https://github.com/rsksmart/rskj/pull/2746#discussion_r1772257679